### DISCUSSION TITLE
Are you happy with the current colour coding in version 7.43?

### DISCUSSION THREAD

**Discussion by fraktal-sb** [XCSoar/XCSoar#1631] [General]
Since version 7.43 airspace colour coding is different. See the following screenshots. Especialy TMZ and RMZ are visualy on the same group as airspace-G. TMZ and RMZ we should not enter without authorization. Airspace-G does not need that. Users could be irritated by the less intensive colour. Unfortunately this colour is badly visibile on different type of screens (e-ink, amoled...). What is your opinion?
Leave it as it is or change it?

![1_AirspaceMap](https://github.com/user-attachments/assets/13fe8a07-9cb6-4b8e-9eae-d7fa46c594d6)
![2_Kobo_Differences](https://github.com/user-attachments/assets/db7d0512-f4e3-44c2-9fa5-2b571b4e8968)
![3_ColourTable](https://github.com/user-attachments/assets/a919054c-3414-4a37-a0f5-8ca27c071949)
![4_DefaultColourTable](https://github.com/user-attachments/assets/d9508e73-e85b-45e6-9770-0453afc3032f)

**Comment by rjfrawley** [comment]
its the task line color (grey) that needs changing, or even better if it was configurable

**Reply by fraktal-sb** [reply]
For mee it's OK now, see https://github.com/XCSoar/XCSoar/issues/1816
